packages/react-components/src/templates/ResearchOutputForm.tsx

```diff
@@ -336,6 +336,7 @@ const ResearchOutputForm: React.FC<ResearchOutputFormProps> = ({
                 const { id } = researchOutput;
                 const savePath = sharedResearch({}).researchOutput({
                   researchOutputId: id,
+                  draftCreated: draftSave && !researchOutputData?.created,
```

Comment:
I've noticed that when we create a draft from a duplicate the toast does not appear because `researchOutputData?.created` is populated when it comes from a duplicate

Reply:
@gabiayako do we know if this is the intended action from duplicate ? Should Research Output B which is duplicated from Research Output A, also duplicate the created date of Research Output A, because Research Output B gets created at that specific moment.

Reply:
> do we know if this is the intended action from duplicate ? 
No, I don't know for sure. It's better to ask Tiff 🤔 I was assuming the banner should appear when a draft is created from an existing output because for me it's a draft like any other one. But it's better to ask Tiff

Reply:
nvm, that's not the issue, we do have the right created date on the new copied RO, but we also have the created date on researchOutputData